### DISCUSSION TITLE
[Feature] Stacking support in Duplicate-handling Utility

### DISCUSSION THREAD

**Discussion by jetpacktuxedo** [immich-app/immich#11735] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sometimes images detected as duplicates are near-duplicates on purpose and I want to stack them instead of deleting some. Right now I could just click "keep all" and then go manually stack them, but it would be convenient to be able to do that while I'm already in the duplicate detection menu.

Here is an example where I want to keep both because it was a burst (names are slightly off because a counter incremented between the pics to indicate the burst):
![image](https://github.com/user-attachments/assets/66b2d141-0b59-4f3f-bb3e-70d46e6b6f90)

Here is an example where I want to keep both because they are a raw+jpg that I want to stack together (+ an actual duplicate of the jpg that I would want to delete)
![image](https://github.com/user-attachments/assets/569fe652-354e-4c72-b7a3-6dbb923ad65f)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10148

Please search for duplicates before opening a FR.
